### ISSUE TITLE
Displaced Menu with compton

### ISSUE THREAD

**Issue by funghetto** [lxqt/lxqt#1706]
##### Expected Behavior
Menu shouldn't be displaced.

##### Current Behavior
[Menu is displaced.](https://i.imgur.com/ylUiRdi.jpg)

##### Steps to Reproduce (for bugs)
1. Enable compton

##### System Information
* Distribution & Version: Gentoo x64
* Kernel: 4.19.27
* Qt Version: 5.11.3-r2
* liblxqt Version: 0.14.1
* Package version: 0.14.1

**Comment by tsujan** [comment]
It seems you're using Kvantum. If so, first upgrade it to its latest release and disable Compton's shadow and blurring for composited Qt menus with this in '~/.config/compton.conf':
```
shadow-exclude = [ "argb && (_NET_WM_WINDOW_TYPE@:a *= 'MENU' || _NET_WM_WINDOW_TYPE@:a *= 'COMBO')" ];
blur-background-exclude = [ "(_NET_WM_WINDOW_TYPE@:a *= 'MENU' || _NET_WM_WINDOW_TYPE@:a *= 'COMBO')" ];
```
You could also disable compositing in Kvantum Manager instead.

Anyhow, the correct place for reporting this is https://github.com/tsujan/Kvantum/issues *if your problem isn't solved.*

**Comment by tsujan** [comment]
FYI:

I tested and saw that the problem doesn't disappear with the above-mentioned changes.

That menu isn't drawn by Kvantum (it's drawn by LXQt stylesheets) but Qt consults Kvantum when showing it and the hassle starts from there: Kvantum moves the menu to compensate for its shadow. However, since the menu isn't drawn by it, there's no shadow; hence the displacement.

I think I've found a way to distinguish such menus in the code. Will test it for a while and then make a commit to work around such issues once for all.
